### ISSUE TITLE
`DynamicContentController` is too aggressive when rehydrating block args

### ISSUE THREAD

**Issue by chrismlusk** [servdhost/craft-asset-storage#45]
We are getting errors in our Sentry dashboard showing a type error where an incorrect argument is passed to `Elements::getElementById()`. The offending URL is `/actions/servd-asset-storage/dynamic-content/get-content`. Looking through the stack trace, I see that these parameters are getting passed to `getElementById()`:
- **elementId**: "near"
- **elementType**: "text"

Digging deeper, the issue is from the [`rehydrateArgs` method in the `DynamicContentController`](https://github.com/servdhost/craft-asset-storage/blob/master/src/controllers/DynamicContentController.php#L73).

The value of the `$a` parameter the `rehydrateArgs` method accepts, per this particular Sentry error log:

```js
{
	degreeCategories: { ... },
	degreeFilters: { ... },
	entry: {
		id: 621293,
		type: craft\elements\Entry,
	},
	// other keys ... 
	inputAttributes: {
		// ...
		id: 'near',
		type: 'text',
	},
}
```

Given that the `rehydrateArgs` method is recursively going through this entire argument object, the fact that my dynamic block has a parameter group with `id` and `type` keys means the controller incorrectly attempts to get an element based on that data.

**Comment by mattgrayisok** [comment]
Oh no! The check to determine whether an element in the content is a candidate for rehydrating is a bit... general, isn't it.

I'll get this updated to more clearly flag rehydration candidates in a way that's less likely to collide with other data.

**Comment by mattgrayisok** [comment]
Fixed and released in 2.6.1 and 3.0.0-beta.10
